### DISCUSSION TITLE
Is there an option to auto reload buffer for external change?

### DISCUSSION THREAD

**Discussion by lzhoucs** [helix-editor/helix#5972] [Q&A] [answered]
Otherwise I have to use `:reload` etc when file is modified externally. Thanks.

**Comment by CptPotato** [comment, marked as answer]
This feature isn't implemented, yet. But as far as I know, some kind of file watching is planned or already in the works, which will allow things like automatic reload.

Here's the issue: #1125

**Comment by lzhoucs** [comment]
Thanks for the reply and information!
